src/components/clusters/wizards/osd/ClusterSettings/CloudProvider/GcpByocFields/GcpByocFields.test.tsx

```
@@ -357,38 +366,43 @@ describe('<GcpByocFields />', () => {
+      expect(screen.queryByRole('button', { name: serviceAccountLabel })).not.toBeInTheDocument();
 
-      // Switch to Service Account
-      await user.click(screen.getByRole('button', { name: serviceAccountLabel }));
+      expect(screen.queryByRole('heading', { name: serviceAccountLabel })).not.toBeInTheDocument();
 
-      expect(
-        await screen.findByRole('checkbox', {
-          name: "I've read and completed all the prerequisites and am ready to continue creating my cluster.",
+      // "Authentication type: Workload Identity Federation" text should be in the document
+      const authTypeElement = screen.getByText(/Authentication type:/i);
+      expect(authTypeElement.parentElement).toHaveTextContent(
```

ESLint

[error] Avoid direct Node access. Prefer using the methods from Testing Library.

(testing-library/no-node-access)